src/main-process/atom-window.js

```diff
@@ -57,9 +58,9 @@ class AtomWindow extends EventEmitter {
 
     Object.defineProperty(this.browserWindow, 'loadSettingsJSON', {
       get: () => JSON.stringify(Object.assign({
-        userSettings: this.atomApplication.configFile.get()
-      }, this.loadSettings)),
-      configurable: true
+        userSettings: this.atomApplication.configFile.get(),
+        projectSettings: this.projectSettings
+      }, this.loadSettings))
```

Comment:
Now that this isn't configurable anymore, could we just remove it and define it as a regular getter method like

```js
constructor () {
  // ...
}

get loadSettingsJSON () {
  return JSON.stringify(/* ... */)
}
```

Reply:
Hmm - aren't you defining this property on `this.browserWindow` rather than on the AtomWindow class? 

I think that Object.define was being used before `configurable: true` was being added.

Reply:
Oh yeah you're right. If we wanted to make it a getter, we'd have to change other code to access it from the atom window instead of the browserWindow. Doable, but no need to do it as part of this PR.